### PR TITLE
Explicitly exclude cutting release for client repo in the release instruction

### PR DESCRIPTION
**Pull request by ZhiminXiang**

### CODE REVIEW

File: RELEASE-LEADS.md [hunk 1]
```diff
@@ -275,7 +277,7 @@ branch of each repository respectively.
 ### Cut `release-x.y` branches of all remaining repositories
 
 After the pin PRs are merged, cut the `release-x.y` branch in each of the
-remaining repositories (except `operator`):
+remaining repositories (except `operator` and `client`):
```
Comment (markusthoemmes) [inline RELEASE-LEADS.md:280]: Shall we add a remark that those two are cut separately by people from the respective working group?

Comment (ZhiminXiang) [reply RELEASE-LEADS.md]: Done

File: RELEASE-LEADS.md [hunk 2]
```diff
@@ -275,7 +277,8 @@ branch of each repository respectively.
 ### Cut `release-x.y` branches of all remaining repositories
 
 After the pin PRs are merged, cut the `release-x.y` branch in each of the
-remaining repositories (except `operator`):
+remaining repositories (except `operator` and `client` as they are cut separately 
+by the respective working group):
```
Comment (mattmoor) [inline RELEASE-LEADS.md:281]: Format markdown:
```suggestion
remaining repositories (except `operator` and `client` as they are cut
separately by the respective working group):
```

**Review by mattmoor — COMMENTED** [review]
Produced via:
  `prettier --write --prose-wrap=always $(find -name '*.md' | grep -v vendor | grep -v .github | grep -v docs/cmd/)`

File: RELEASE-LEADS.md (hunk 2 above)
Comment (ZhiminXiang) [reply RELEASE-LEADS.md]: Done

**Review by mattmoor — COMMENTED** [review]
/lgtm
/approve
/hold

HOLD in case there were other outstanding comments I didn't spot!

**Review by tcnghia — COMMENTED** [review]
/lgtm

Thanks for updating.

**Review by rhuss — APPROVED** [review]
thanks !

/lgtm

**Pull request merged**